include/maths/analytics/CBoostedTreeLoss.h

```diff
@@ -991,7 +1059,36 @@ class MATHS_ANALYTICS_EXPORT CPseudoHuber final : public CLoss {
     bool acceptRestoreTraverser(core::CStateRestoreTraverser& traverser) override;
 
 private:
-    double m_Delta;
+    std::size_t m_NumberClasses;
+};
+
+class MATHS_ANALYTICS_EXPORT CMultinomialLogisticSubsetLoss final
```

Comment:
Can you please add a comment about the purpose of this class?

Reply:
Done in [48903cd](https://github.com/elastic/ml-cpp/pull/2395/commits/48903cd58633aaacd60ad6d7ad64b7cf9acaa034).